docs/source/1.0/guides/building-models/build-config.rst

```diff
@@ -1446,14 +1446,14 @@ Smithy models contained within the JAR are copied as a source model while the
 JAR itself is not copied. If there are no source models, an empty manifest is
 created.
 
-When applying a projection, a new model file is created that contains only
-the shapes, trait definitions, and metadata that were defined in a source
+When building a different projection, a new model file is created that contains
+only the shapes, trait definitions, and metadata that were defined in a source
 model *and* all of the newly added shapes, traits, and metadata.
```

Comment:
Would be nice to clarify what "newly added" means here. I thought that was the main reason for the PR.